api/controllers/auth.js

```diff
@@ -44,6 +50,48 @@ const registration = async (req, res, next) => {
   }
 };
 
+const reset = async (req, res, next) => {
+  const { email } = req.body;
+  try {
+    const v = await emailSchema.validate({ email });
+    const a = v && (await services.resetPasswordRequest({ email }));
+    let status = a.status ? a.status : 200;
+    let response = a.messageId ? 'Mail was sent' : a.message;
+    res.status(status).send(response);
+  } catch (e) {
+    next(e);
+  }
+  //
+};
+
+const resetApprovementPassword = async (req, res, next) => {
+  const { linkId } = req.params;
+  try {
+    const UserId = await services.resetPasswordApprove({
+      linkId: decodeURIComponent(linkId),
```

Comment:
It is not need to wrap all code in try/catch blocks

Reply:
Try/catch is a useful code that helps to not break (fell down) application.

Reply:
Try/catch block helps us to catch unpredictable errors. In our case all errors are predictable.